exec/java-exec/src/main/java/org/apache/drill/exec/server/options/SystemOptionManager.java

```diff
@@ -346,17 +347,63 @@ public void deleteAllOptions(OptionType type) {
     }
   }
 
-  public void populateDefaultValues() {
-
+  public static CaseInsensitiveMap<OptionValue> populateDefualtValues(Map<String, OptionDefinition> definitions, DrillConfig bootConfig) {
```

Comment:
`populateDefualtValues` --> `populateDefaultValues` (that is "ua" --> "au").

Reply:
Done